odl/discr/discr_ops.py

```diff
@@ -36,10 +35,11 @@ def __init__(self, domain, range, interp):
 
         Parameters
         ----------
-        domain : `DiscretizedSpace`
+        domain : `DiscreteLp`
```

Comment:
Hm, I was skeptical on first thought, but on second thought, since this space is so ubiquitous we can as well simplify the name. I'll go for `DiscretizedSpace`. :+1: